### PR TITLE
SDC-1719 feature/github-actions

### PR DESCRIPTION
**Pull request by gaetancollaud**
SDC-1719

### CODE REVIEW

File: .github/workflows/release.yaml.disabled [hunk 1]
```diff
@@ -0,0 +1,166 @@
+name: Java build EC2
```
Comment (spike83) [inline .github/workflows/release.yaml.disabled:1]: is this file still needed?

Comment (gaetancollaud) [reply .github/workflows/release.yaml.disabled:1]: No I will remove it.

tags are catched from the "push" event from the other pipeline

**Review by RuiTeixeira — APPROVED** [review]
